Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -306,6 +308,22 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
       self.pledgeDisclaimerViewDidTapLearnMoreSignal.mapConst(.trust)
     )
 
+    self.configureEstimatedShippingView = Signal.combineLatest(project, baseReward, selectedShippingRule)
+      .map { project, reward, shippingRule in
+        guard let rule = shippingRule else { return ("", "") }
```

Comment:
Does it make sense to make this Signal return an Optional and return a nil here, rather than a pair of empty strings? Answer might be no, just safety checking